specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/examples/CreateIdentity.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2021-03-07",
+    "content-type": "application/json",
+    "endpoint": "https://my-resource.communication.azure.com",
+    "body": {
+      "createTokenWithScopes": [
+        "chat"
+      ]
+    }
+  },
+  "responses": {
+    "200": {
```

Comment:
See other comment, I think we should use 201

Reply:
Updated